test/e2e-common.sh

```diff
@@ -21,6 +21,13 @@ source $(dirname $0)/../vendor/knative.dev/test-infra/scripts/e2e-tests.sh
 # If gcloud is not available make it a no-op, not an error.
 which gcloud &> /dev/null || gcloud() { echo "[ignore-gcloud $*]" 1>&2; }
 
+# Let sed work on mac.
+if [ "$(uname)" == "Darwin" ]; then
+  sed=gsed
+elif [ "$(expr substr $(uname -s) 1 5)" == "Linux" ]; then
+  sed=sed
+fi
```

Comment:
If we aren't on either (or the script just can't figure it out), should this silently pass?

Reply:
Right now we only have this error on MAC, so I removed the `elif`. I don't think we need to fail it here since `sed` is the only command we've found to be system dependent, and it should be working in most cases (except Windows)...